addons/api/mirage/config.js

```diff
@@ -514,6 +514,14 @@ export default function () {
   this.del('/credential-libraries/:id');
   this.patch('/credential-libraries/:id');
 
+  // managed-groups
+  this.get(
```

Comment:
For now just adding the `get` method we will need for the list. Later on I will add more methods.